src/selectors/entities/users.test.js

```diff
@@ -186,12 +192,50 @@ describe('Selectors.Users', () => {
                             [userId]: {id: userId, username: 'user', first_name: 'First', last_name: 'Last', notify_props: notifyProps},
                         },
                     },
+                    groups: {
+                        groups: {},
+                    },
                 },
             };
 
             assert.deepEqual(Selectors.getCurrentUserMentionKeys(state), [{key: 'First', caseSensitive: true}, {key: '@user'}]);
         });
 
+        it('groups', () => {
+            const userId = '1234';
+            const notifyProps = {
+                first_name: 'true',
+            };
+            const state = {
+                entities: {
+                    users: {
+                        currentUserId: userId,
+                        profiles: {
+                            [userId]: {id: userId, username: 'user', first_name: 'First', last_name: 'Last', notify_props: notifyProps},
+                        },
+                    },
+                    groups: {
+                        groups: {
+                            test1: {
+                                display_name: 'I_AM_THE_BEST!',
+                                allow_reference: true,
+                            },
+                            test2: {
+                                display_name: 'Do you love me?',
```

Comment:
are we operating under the assumption that groups can have spaces in between them?

Reply:
> are we operating under the assumption that groups can have spaces in between them?

Just a test, I didn't really think of that. I just made a random display name. I can change it to a  non-spaced display name if it's better.

Reply:
I've made the display names non-spaced.

Reply:
It was more of a question - since my backend code will not be able to parse a group display name like this

Reply:
> It was more of a question - since my backend code will not be able to parse a group display name like this

@fm2munsh 
I agree that it shouldn't be spaced cause that can be confusing. I don't think we can have groups with spaces in them and it's better to align the tests more with how they would actually be.